packages/sw-background-sync-queue/demo/index.html

```diff
@@ -0,0 +1,28 @@
+<html>
+	<head></head>
+	<body>
+		<div>Hello dear</div>
+		<button onclick="putReqInQ('1')">send req</button>
+		<button onclick="putReqInQ('2')">send req</button>
+		<button onclick="putReqInQ('3')">send req</button>
+		<button onclick="putReqInQ('4')">send req</button>
+		<button onclick="putReqInQ('5')">send req</button>
+		<script>
+			var _reg;
```

Comment:
it appears `_reg` gets assigned the response from the Service Worker registration but other than the assignment it doesn't appear to get used. I think it's fine to just `console.log` and skip that part.